pkg/uroot/util/unixflag.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package util
```

Comment:
How about changing the package name from "util" to something like "uflag" or "unixflag"? My concern is that the package "util" may become a catch-all for functions that have nothing to do with each other.

Reply:
sure, I will go with unixflag.

The name util is historical, and goes back almost to the beginning. that name is now discouraged.